src/test/java/Problem2Test.java

```diff
@@ -7,13 +7,25 @@
     public void testRemoveDuplicates() {
         int[][] inputs = {
                 // DO NOT REMOVE OR CHANGE THE GIVEN TEST CASES!
-                {0, 0, 1, 1, 1, 2, 2, 4, 4, 6}
+                {0, 0, 1, 1, 1, 2, 2, 4, 4, 6},
                 // add more test cases
+                {0,1,2,3,4,5,6,7},
+                {0,0,0},
+                {0,1,2,2,2,3,3,4,5,6,7,8,8,9},
+                {0,1,1,1,1,2},
+                {0,1,2,3,4,5,6,7,8,9,10,11,11,11,11,11,11,11},
+                {0,0,2,2,3,6,7,8,8,9,10,12,15}
         };
 
         int[][] expects = {
-                {0, 1, 2, 4, 6}
+                {0, 1, 2, 4, 6},
                 // add more test cases
+                {0,1,2,3,4,5,6,7},
+                {0},
+                {0,1,2,3,4,5,6,7,8,9},
+                {0,1,2},
+                {0,1,2,3,4,5,6,7,8,9,10,11},
+                {0,2,3,6,7,8,9,10,12,15}
```

Comment:
great test. let's add more cases for the edge cases, such as null array and empty array.